stratum/hal/lib/common/hal.cc

```diff
@@ -329,6 +337,15 @@ ::util::Status Hal::RegisterSignalHandlers() {
     }
     old_signal_handlers_[s] = h;
   }
+  // Create the pipe to transfer signals.
+  int pipe_fds[2];
+  CHECK_RETURN_IF_FALSE(pipe(pipe_fds) == 0) << "Could not create pipe.";
```

Comment:
Could we use a more descriptive message? "pipe for signal handling"?
```suggestion
  CHECK_RETURN_IF_FALSE(pipe(pipe_fds) == 0) << "Could not create pipe for signal handling.";
```